docs/beta/plugins/discord.md

```diff
@@ -1,5 +1,248 @@
+
+  SourceCred can be configured to mint different amounts of Cred for different emojis . This is done by setting custom reaction weights in the Discord plugin configuration file: `config/plugins/sourcecred/discord/config.json`. 
+  
+  
+  If you want to change the weight for a default reaction, just put the reaction's literal emoji in `config.json`. For example, if you wanted the 💜 emoji to have a weight that is 5 times a regular reaction, add the line `"💜": 5` to the `config.json`, as shown below. 
+  
+  ```
+  {
+  "guildId": "742885338065666150",
+  "reactionWeights": {
+    "👍": 1,
+    "💜": 5,
+    "sourcecred:678399364418502669": 10,
+  }
+}
+```
+  If you want to set a custom weight for a custom emoji, you'll first need to get the custom emoji ID. You can get this by right clicking on the custom emoji and clicking 'Copy Link'. This will give you a URL to an image of the emoji. The numberic substring in the URL is the emoji ID. For instance, below is the URl for a custom sourcecred emoji called `:sourcecred_pink:`.
+  
+  `https://cdn.discordapp.com/emojis/745438325086879794.png?v=1`
+  
+  The emoji ID is `745438325086879794`. The reaction weight is then specified in `config.json` using the following format: `$EMOJI_NAME:$EMOJI_ID`. So if we want to have `:sourcecred_pink:` be 20 times a regular reaction, we would add the below line to `config.json`.
+  
+  `"sourcecred_pink:745438325086879794": 20`
+  
+  
+
+
+[Yarn]: https://classic.yarnpkg.com/
+
+## Status and Caveats
+
+{SB: below copied and pasted from Discourse draft for inspiration. Need to create for discord plugin}
+
+```
+The Discourse plugin is currently in Beta. It assigns Cred scores that are reasonable and
+robust for a [trust level]{SB: needs link} 3 community.
+
+Currently, there are two general approaches to minting Cred you can take with the
+Discourse pluginL activity-minted and like-minted Cred (or a hybrid of the two).
+Activity-minted Cred means Cred would be minted for each new topic or post
+created. While like-minted Cred mints new Cred for each like given.
+
+The plugin defaults to using like-minted Cred. For most communities this will
+be the better approach. Unlike activity-minted Cred, like-minted Cred allows people to
+validate contributions. People soon learn their likes signal "this is valuable
+and I'd like to see more of this", which incentivizes the creation of
+high-quality posts.
+
+The caveat is that it creates some "popularity contest" dynamics, where memes
+and/or heavily promoted posts might receive more likes than makes sense for the
+relative value they've added. It's something which would be easy to game, making it
+less suitable for lower trust levels for the time being.
+
+{SB: the below para wasn't in the original docs, but is just me speculating based
+on past convos. We may want to remove if we can't provide enough
+guideance for it to be useful}
+Another caveat is that when a Discourse forum is very new, and does not have 
```

Comment:
Some references to Discourse here. Was this meant to stay?

Reply:
Not meant to stay. Just to give a reference for whoever writes the Status and Caveats section for Discord. Do you have any issues in mind that a user should know here? More easily gamable than other plugins? Known bugs? Design quirks?

Reply:
Nah. Just left for reference/inspiration to whoever wanted to write the Status and Caveats. Have removed.